packages/core/src/Banner/BannerContent/BannerContent.tsx

```diff
@@ -79,7 +79,7 @@ export const HvBannerContent = forwardRef<
       ref={ref}
       id={id}
       variant={variant}
-      showClose
+      showClose={onClose !== undefined}
```

Comment:
without the icon, some Banners are becoming too small, and some actions are positioned incorrectly